src/state_machine_client/src/lib.rs

```diff
@@ -189,6 +224,16 @@ impl StateMachine {
         self.call_state_machine(Request::AdvanceTime(duration))
     }
 
+    pub fn tick(&self) {
+        self.call_state_machine(Request::Tick)
+    }
+
+    pub fn run_until_completion(&self, max_ticks: u64) {
+        self.call_state_machine(Request::RunUntilCompletion(RunUntilCompletionArg {
+            max_ticks,
+        }))
+    }
```

Comment:
I'm confused, is this used anywhere?

Reply:
Not yet. But it will be used by the integration test for the archive pull integration.